test/e2e/lib/components/nav-bar-component.js

```diff
@@ -41,14 +41,6 @@ export default class NavBarComponent extends AsyncBaseContainer {
 	async clickMySites() {
 		const mySitesSelector = by.css( 'header.masterbar a.masterbar__item' );
 		await driverHelper.clickWhenClickable( this.driver, mySitesSelector );
-		await driverHelper.isEventuallyPresentAndDisplayed(
-			this.driver,
-			by.css( '.sidebar__menu-wrapper' )
-		);
-		return await driverHelper.isEventuallyPresentAndDisplayed(
```

Comment:
Out of curiosity, why are they unnecessary? It seems like we are just making sure that the menu loads in before continuing tests?

Reply:
These kinds of waits are most of the time unnecessary because they're followed by a `clickWhenClickable` call (100% time with any menu element). The `clickWhenClickable` helper will wait until the element is clickable, which is the same as waiting for the menu to be loaded and then calling the click helper. It's better to use the click helper only because we're waiting for the very element we want to interact with. In this case, we were even waiting additionally for some other element (`.is-goup-sites`), whose relation to the menu loading state seems to be unclear. Does that make sense?

Reply:
Makes sense!